### PR TITLE
Swift

### PR DESCRIPTION
**Pull request by morganchen12**
Adds Swift 3.0 version of objc stuff.

@ianbarber please review

### CODE REVIEW

File: ios/swift/FirebaseAnalyticsWeb.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -0,0 +1,674 @@
...
+			<key>name</key>
+			<string>[CP] Check Pods Manifest.lock</string>
+			<key>outputPaths</key>
+			<array/>
+			<key>runOnlyForDeploymentPostprocessing</key>
+			<string>0</string>
+			<key>shellPath</key>
+			<string>/bin/sh</string>
+			<key>shellScript</key>
+			<string>diff "${PODS_ROOT}/../Podfile.lock" "${PODS_ROOT}/Manifest.lock" &gt; /dev/null
```
Comment (ianbarber) [inline ios/swift/FirebaseAnalyticsWeb.xcodeproj/project.pbxproj]: Deintegrate

File: ios/swift/FirebaseAnalyticsWeb/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+class ViewController: UIViewController, WKScriptMessageHandler {
+
+  private var webView: WKWebView!
+  private var projectURL: URL!
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    // Get the hosted site url from the info.plist file.
```
Comment (ianbarber) [inline ios/swift/FirebaseAnalyticsWeb/ViewController.swift]: From the GoogleService-Info.plist, else is a bit confusing

**Review by ianbarber — APPROVED** [review]
Looks good, one nit, and i think it needs deintegrating.

**Comment by samtstern** [comment]
Merging since @morganchen12 has no power here.

**Pull request merged**